src/python/foglamp/configuration_manager.py

```diff
@@ -29,7 +31,8 @@
 
 _valid_type_strings = ['boolean', 'integer', 'string', 'IPv4', 'IPv6', 'X509 certificate', 'password']
```

Comment:
This is what I requested to @ashwinscale after seeing in requirement doc for https://github.com/foglamp/FogLAMP/pull/70:

<img width="558" alt="types" src="https://user-images.githubusercontent.com/1974003/28339161-cdead076-6c28-11e7-8c0b-13e5dc1bd0ff.png">
 please confirm if we should fix here (and update the doc)?